# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tibit/qiv/00-config.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ur qiv-1.8/Makefile patched-qiv-1.8/Makefile
--- qiv-1.8/Makefile	2001-06-10 14:35:06.000000000 +0200
+++ patched-qiv-1.8/Makefile	2003-02-21 20:38:08.000000000 +0100
@@ -4,7 +4,7 @@
 ######################################################################
 
 # Directory where qiv will be installed under.
-PREFIX = /usr/local
+PREFIX = /usr
 
 # Font to use for statusbar in fullscreen mode
 STATUSBAR_FONT = "fixed"
@@ -14,10 +14,10 @@
 CURSOR = 84
 
 # Should image be centered on screen by default? 1=yes, 0=no.
-CENTER = 1
+CENTER = 0
 
 # Should images be filtered by extension? 1=yes, 0=no.
-FILTER = 1
+FILTER = 0
 
 # This sets the file extentions to filter on (other file types will be
 # skipped.) It should reflect whatever is compiled into imlib.
@@ -28,7 +28,7 @@
 
 # This program will be run on the manual page after it is installed.
 # If you don't want to compress the manpage, change it to 'true'.
-COMPRESS_PROG = gzip -9f 
+COMPRESS_PROG = true
 
 ######################################################################
 
@@ -114,11 +114,6 @@
 	@echo "Installing..."
 	install -s -m 0755 $(PROGRAM) $(PREFIX)/bin
 	install -m 0644 $(PROGRAM).1 $(PREFIX)/man/man1
-	$(COMPRESS_PROG) $(PREFIX)/man/man1/$(PROGRAM).1
-	@if ./qiv -o white -f ./intro.jpg ; \
-	then echo "-- Test Passed --" ; \
-	else echo "-- Test Failed --" ; \
-	fi
 
 install-xscreensaver: install
 	@echo "#!/bin/sh" > $(SS_PROG)
